client/src/components/UpcomingEventsCard/UpcomingEventsCard.css

```diff
@@ -0,0 +1,28 @@
+.UpcomingEventsCard {
+    width: 17vw;
+    height: 14vh;
+    border-radius: 3px;
+    background-color: var(--pink);
+    filter: drop-shadow(0px 4px 4px rgba(0, 0, 0, 0.25));
+    color: var(--white);
+    display: flex;
+    flex-direction: column;
+    padding: 10px;
+    text-decoration: none;
+}
+
+.UpcomingEventsCard__name {
+    font-family: var(--default-font);
+    font-size: 2vw;
+}
+
+.UpcomingEventsCard__date,
+.UpcomingEventsCard__time,
+.UpcomingEventsCard__location {
+    font-family: var(--paragraph-font);
+    font-size: 1.2vw;
+}
+
+.UpcomingEventsCard__location {
+    padding-top: .8vh;
+}
```

Comment:
You may want to add some padding to the bottom. The location is going over the bottom edge of the card.